### DISCUSSION TITLE
Dual Gun Setup?

### DISCUSSION THREAD

**Discussion by isimpson20925** [TeamOpenFIRE/OpenFIRE-Firmware#9] [Q&A]
I have two guns setup but I'm having issues getting them to register as two different devices in the emulators. Is there any kind of guide on setting this up?

**Comment by SeongGino** [comment]
> I have two guns setup but I'm having issues getting them to register as two different devices in the emulators. Is there any kind of guide on setting this up?

Everything regarding multiple guns is already provided in the documentation, and has a dedicated section in the app to configure this. Make sure both of your devices have at least different identifiers - or just choose the player number presets in the app.

You can redistribute a gun's start and select button mappings on the fly with serial commands as well - again, it's all provided in the documentation.
